Source/WebCore/mathml/MathMLOperatorElement.cpp

```diff
@@ -212,42 +213,60 @@ void MathMLOperatorElement::childrenChanged(const ChildChange& change)
 
 static std::optional<MathMLOperatorDictionary::Flag> attributeNameToPropertyFlag(const QualifiedName& name)
 {
-    if (name == accentAttr)
+    switch (name.nodeName()) {
+    case AttributeNames::accentAttr:
         return Accent;
-    if (name == fenceAttr)
+    case AttributeNames::fenceAttr:
         return Fence;
-    if (name == largeopAttr)
+    case AttributeNames::largeopAttr:
         return LargeOp;
-    if (name == movablelimitsAttr)
+    case AttributeNames::movablelimitsAttr:
         return MovableLimits;
-    if (name == separatorAttr)
+    case AttributeNames::separatorAttr:
         return Separator;
-    if (name == stretchyAttr)
+    case AttributeNames::stretchyAttr:
         return Stretchy;
-    if (name == symmetricAttr)
+    case AttributeNames::symmetricAttr:
         return Symmetric;
+    default:
+        break;
+    }
     return std::nullopt;
 }
 
 void MathMLOperatorElement::attributeChanged(const QualifiedName& name, const AtomString& oldValue, const AtomString& newValue, AttributeModificationReason attributeModificationReason)
 {
-    if (name == formAttr) {
+    if (auto flag = attributeNameToPropertyFlag(name))
+        m_properties.dirtyFlags |= flag.value();
```

Comment:
Could just move the code down into a single switch statement. Not sure that sharing `m_properties.dirtyFlags |=` is worth it.

Reply:
I like it. I'll look into it.